source/PluginDev/Assets/GooglePlayGames/Editor/GPGSPostBuild.cs

```diff
@@ -68,13 +68,13 @@ public static void OnPostprocessBuild(BuildTarget target, string pathToBuiltProj
             #if NO_GPGS
 
             string[] filesToRemove = {
-                "Libraries/Plugins/iOS/GPGSAppController.mm",
+                "Libraries/Plugins/IOS/GPGSAppController.mm",
```

Comment:
I think this depends on the version of Unity?  mine are mixed cased.  Can you add both cases to the list or do case insensitive comparison when cleaning up the project?